spec/requests/shops_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe "Shops", type: :request do
+  describe "GET /shop" do
+    it "returns http success" do
+      get "/shops/shop"
+      expect(response).to have_http_status(:success)
+    end
+  end
+
```

Comment:
?
empty space.

Reply:
all is correct empty line in the end is mast be

Reply:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file